### PR TITLE
Profiler and reboot for admins with +SERVER/+DEBUG, client skin stuff.

### PR DESCRIPTION
**Pull request by PJB3005**
Also yes I had to drop control freak too because BYOND is moronic.

BYOND Admin config stuff ported from TG.

Closes #14245 because BYOND is retarded.

:cl:
* rscadd: All admins with +DEBUG or +SERVER now get profiler access, and having +SERVER grants you hard reboot access. There are buttons for this under the "Server" dropdown.
* rscadd: Renamed the "File" dropdown to "Client"
* rscadd: There is now a reconnect button under the "Client" drop down menu.

### CODE REVIEW

File: vgstation13.dme [hunk 1]
```diff
@@ -2162,7 +2163,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```
Comment (Probe1) [inline vgstation13.dme]: Thanks Doc

**Review by Probe1 — CHANGES_REQUESTED** [review]

File: vgstation13.dme (hunk 1 above)
Comment (PJB3005) [reply vgstation13.dme]: I'm a genius I'm telling you.

**Comment by Intigracy** [comment]
Reboot isn't logged though, pls log.

**Comment by PJB3005** [comment]
@Intigracy if you'd read the diff you'd see I added logging.

**Review by Probe1 — APPROVED** [review]

**Pull request merged**